packages/teams-js/test/public/people.spec.ts

```diff
@@ -26,270 +26,233 @@ describe('peoplePicker', () => {
       app._uninitialize();
     }
   });
+  const allowedContexts = [FrameContexts.content, FrameContexts.task, FrameContexts.settings];
+  const result = [
+    {
+      objectId: '5842943a-aa5a-470a-bfdc-7311b9988962',
+      displayName: 'Sonal Jha',
+      email: 'sojh@m365x347208.onmicrosoft.com',
+    } as people.PeoplePickerResult,
+  ];
+  const input = {
+    title: 'Title',
+    openOrgWideSearchInChatOrChannel: true,
+  };
+  describe('peoplePicker', () => {
+    /**
+     * People Picker tests
+     */
+    it('should not allow selectPeople calls before initialization', async () => {
+      expect(() => people.selectPeople()).toThrowError('The library has not yet been initialized');
+    });
+    Object.values(FrameContexts).forEach(context => {
+      if (allowedContexts.some(allowedContext => allowedContext === context)) {
+        return;
```

Comment:
Instead of using return can you please put the test cases where we initialize it using the allowed contexts.